scripts/generate-bakefiles.js

```diff
@@ -73,18 +76,18 @@ function generateCombinations(arr) {
 	// keep track index in each of the n arrays
 	const indices = new Array(n).fill(0);
 
-	while (true) {
-		let tmp = [];
+	for (;;) {
```

Comment:
This seems like a hack to just avoid the warning around `while(true)` without actually solving the intent - I'd prefer to just add a silence for the linting error so that we keep the more standard/easier to spot `while(true)` format